css/index.css

```diff
@@ -137,10 +182,109 @@ footer nav {
     justify-content: space-between;
     align-items: center;
     padding: 20px 2%;
-    font-size: 14px;
+    font-size: .8rem;
 }
 
 footer nav a {
     color: white;
     text-decoration: none;
+}
+
+/* Ipad */
+@media (max-width: 768px) {
+   
+    header {
+        display: block;
+    }
+    nav {
+        margin-left: 8.4%;
+        margin-top: 20px;
+    }
+
+    .lambda-logo {
+        margin-left: 38.5%;
+        margin-top: 22px;
+    }
+    .text-container h2 {
+        font-size: 1rem;
+    }
+
+    .text-container h2 p {
+        font-size: 14px;
+    }
+    .middle-content h2{
+        font-size: 1rem;
+    }
+
+    .top-content .text-container p {
+        font-size: 14px;
+    }
+    .middle-content .boxes .box {
+        height: 84px;
```

Comment:
The tablet-design.png reference image has these boxes still at `100px` by `100px`. No need to reduce their height.